docs/reference/index-modules/similarity.asciidoc

```diff
@@ -32,7 +32,7 @@ PUT /index
                 "normalization" : "h2",
                 "normalization.h2.c" : "3.0"
               }
-            }
+           }
```

Comment:
Github won't allow me to comment on the line, but do you mind fixing the indentation on line 27 (` "similarity"`) as well?

It looks like it uses tab/5 spaces.